docs/QUICKSTART.md

```diff
@@ -1,76 +1,208 @@
+    kind-worker2         Ready    <none>          44m   v1.24.0
+    ```
+
+    List the images then filter for nginx:
+
+    ``` shell
+    $ docker exec kind-worker ctr -n k8s.io images list | grep nginx
+    docker.io/library/nginx:1.14.2                                                                  application/vnd.docker.distribution.manifest.list.v2+json sha256:f7988fb6c02e0ce69257d9bd9cf37ae20a60f1df7563c3a2a6abe24160306b8d 42.6 MiB  linux/386,linux/amd64,linux/arm/v7,linux/arm64/v8,linux/ppc64le,linux/s390x  io.cri-containerd.image=managed 
+    docker.io/library/nginx@sha256:f7988fb6c02e0ce69257d9bd9cf37ae20a60f1df7563c3a2a6abe24160306b8d application/vnd.docker.distribution.manifest.list.v2+json sha256:f7988fb6c02e0ce69257d9bd9cf37ae20a60f1df7563c3a2a6abe24160306b8d 42.6 MiB  linux/386,linux/amd64,linux/arm/v7,linux/arm64/v8,linux/ppc64le,linux/s390x  io.cri-containerd.image=managed 
+    ```
+
+5. Create an ImageList
+
+    Create an [ImageList](../test/e2e/test-data/eraser_v1alpha1_imagelist.yaml) and specify the images you would like to remove. In this case, the image `docker.io/library/nginx:1.14.2` will be removed.
+
+      ```bash
+      cat <<EOF | kubectl apply -f -
+      apiVersion: eraser.sh/v1alpha1
+      kind: ImageList
+      metadata:
+        name: imagelist
+      spec:
+        images:
+          - docker.io/library/nginx:1.14.2
+      EOF
+      ```
+
+    > ImageList is a cluster-scoped resource and must be called `imagelist`. You can specify `"*"` for all non-running images.
+
+    Creating an `imagelist` should trigger an [ImageJob](api/v1alpha1/imagejob_types.go) that will deploy Eraser pods on every node to perform the removal given the list of images.
+
+    ```bash
+    $ kubectl get pods -n eraser-system
+    eraser-system        eraser-controller-manager-55d54c4fb6-dcglq   1/1     Running   0          9m8s
+    eraser-system        eraser-kind-control-plane                    1/1     Running   0          11s
+    eraser-system        eraser-kind-worker                           1/1     Running   0          11s
+    eraser-system        eraser-kind-worker2                          1/1     Running   0          11s
+    ```
+
+    Pods will run to completion and the images will be removed.
+
+    ```bash
+    $ kubectl get pods -n eraser-system
+    eraser-system        eraser-controller-manager-6d6d5594d4-phl2q   1/1     Running     0          4m16s
+    eraser-system        eraser-kind-control-plane                    0/1     Completed   0          22s
+    eraser-system        eraser-kind-worker                           0/1     Completed   0          22s
+    eraser-system        eraser-kind-worker2                          0/1     Completed   0          22s
+    ```
+
+    The `imagelist` custom resource status field will contain the status of the last job.
+
+    ```bash
+    $ kubectl describe ImageList imagelist
+    ...
+    Status:
+      Failed:     0
+      Success:    3
```

Comment:
We should make it clear that the Success count indicates the number of nodes that the Eraser agent was run successfully on,  NOT the number of non-running images that were removed.  There is currently no way to determine what was removed other than manual inspection of each node.

Reply:
updated in https://github.com/Azure/eraser/pull/186/commits/af9467fa5f8a1cb98718b570ce22b865379977a7